action.yml

```diff
@@ -60,6 +60,14 @@ inputs:
   additional-ports:
     description: Ports the client needs to access the application in addition to the one in the app URL, as a list of comma-separated values
     required: false
+  base-sha:
+    description: |
+      The base commit SHA to compare against. This can usually be inferred automatically by Meticulous, so typically you do not need to set this.
```

Comment:
Nit: it might be worth mentioning that this will override the sha inferred by Meticulous. The description kind of makes it seem like this input is a fallback if the sha can't be inferred.